packages/palette/src/elements/Cards/SmallCard.tsx

```diff
@@ -1,11 +1,83 @@
 import React from "react"
+import { space } from "../../helpers"
+import { Box } from "../Box"
+import { Flex } from "../Flex"
+import { Image } from "../Image"
+import { Spacer } from "../Spacer"
+import { Sans } from "../Typography"
+import { CardTag } from "./CardTag"
 import { SmallCardProps } from "./SmallCard.shared"
 
 /**
  * `SmallCard` is a card with a layout one square image on the left,
  * one tall or two square images on the right, and text for title and subtitle
  * at the bottom.
  */
-export const SmallCard: React.FC<SmallCardProps> = () => {
-  return null
+export const SmallCard: React.FC<SmallCardProps> = ({
+  images,
+  title,
+  subtitle,
+  tag,
+}) => {
+  return (
+    <Box width="100%" position="relative">
+      <Box width="100%" height="0" pt="67%" position="relative">
```

Comment:
Was not sure if there is a better was to implement `ratio` in web views. But that seems to work - so rolling with it here. Suggestions are welcome though:)

Reply:
I did try the same thing for eigen here https://github.com/artsy/eigen/pull/3457/commits/8fdd1b2e6cdb87cdd0d47f2a7a3518710d435154#diff-89e2903a44c036391e4b31d92dba65faR8 bt thankfully RN has `aspectRatio`. I think for web this is a common way. If we end up needing this more, we could add an extra Layout component in the future.

Reply:
This works. I'm going to add an [aspect ratio box component](https://github.com/auspices/eos/blob/master/src/AspectRatioBox/AspectRatioBox.tsx) in here soon. 🤷

Reply:
Refactor scheduled for the future (TM) :)